src/Client.php

```diff
@@ -627,9 +628,11 @@ public function listGroups(ListGroupsQuery $query): array {
         $bodyAsXml = simplexml_load_string((string) $response->getBody());
 
         if ((string) $bodyAsXml->Result === 'Failed') {
+
```

Comment:
NIT our style guide indicates code blocks should not start with an empty line

Reply:
@tomegantcs  good catch.  I think there was an interim step where where I was troubleshooting something where I added code, then removed it, and the newline got leftover as a vestigial organ of the change. I'll remove it before merge.